### PR TITLE
Move all test/tools.py fixtures to conftest.py file

### PR DESCRIPTION
**Pull request by dorothykiz1**
xref #1030

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -36,6 +36,7 @@ exclude = setup.py,
         test/test_continuous.py,
         test/test_util.py,
         test/test_find.py,
+        test/tools.py,
```
[the remark this answers is not in the document]
Comment (dorothykiz1) [reply setup.cfg:39]: I added it as it was complaining about flake8 issues, I remember you had advised me not to mix issues.
Could my approach be wrong or i fix the unused import flake8 issues in the file?

[the remark this answers is not in the document]
Comment (dorothykiz1) [reply setup.cfg:39]: Thanks for this,let me fix this

**Comment by dorothykiz1** [comment]
@datapythonista this PR is ready for your review again

**Comment by datapythonista** [comment]
Sorry, this has conflicts with your other PR I just merged. Can you resolve them please?

**Comment by datapythonista** [comment]
Tests are failing. Looks like you removed the decorator making `basic_html` a fixture when moving things.

**Comment by dorothykiz1** [comment]
Thanks @datapythonista I have worked on this,you can check again on green

**Comment by datapythonista** [comment]
Thanks @dorothykiz1, very nice!

**Pull request merged**